functional-samples/sample.co2meter/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "CO2 meter extension",
+  "description": "An extension that connects to a CO2  meter using WebHID extension service workers",
+  "version": "1.0",
+  "manifest_version": 3,
+  "background": {
+    "service_worker": "background.js",
+    "type": "module"
+  },
+  "action": {
+    "default_icon": {
+      "32": "images/icon32.png"
+    },
+    "default_title": "Click me",
+    "default_popup": "popup.html"
+  },
+  "permissions": ["scripting","alarms"],
```

Comment:
Do you need the alarms and scripting permissions?

Reply:
No, these are left-over, cleared.